app/services/projects_json_export_service.rb

```diff
@@ -182,6 +182,6 @@ def tiny_mce_file_json(asset)
   end
 
   def asset_url(asset)
-    ENV['MAIL_SERVER_URL'] + Rails.application.routes.url_helpers.asset_file_url_path(asset)
+    Rails.application.routes.url_helpers.asset_file_url_url(asset)
```

Comment:
`_url_url` probably a mistake?

Reply:
no, it is a combination of controller action name + _url URL helper suffix